README.org

```diff
@@ -29,3 +29,14 @@ For more information check the manuals.
 Return non-nil if ELT is an element of LIST.  Comparison done with ‘equal’.
 The value is actually the tail of LIST whose car is ELT.
 #+end_quote
+** ~when-let~
```

Comment:
I was thinking that it might be better to have `if_let` in Nim to mimic `if-let*` in Emacs-Lisp. That way we can have an `else:` condition too. I know that I started off with when-let suggestion to begin with, but this thought came up very late.

Also `when` has a "compile-time" meaning in Nim, so we rather have `if_let` instead of `when_let` as it can anyways do what the latter does.

Reply:
Agreed. Originally I thought there'd be some way to make use of `when`. 

I'll change it to `if_let` and make the `else` branch optional. So we get both at the same time.